coding-concepts.js

```diff
@@ -21,16 +21,16 @@ const greeter = (name) => {
 }
 // console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: this will return "Hello LEARN Student"
+// b) Verify and explain: my answer was wrong, it returned undefined. this is because ${} is used to add expressions and in this case "LEARN Student is not an expression but instead just a string.
```

Comment:
Remember every function needs a return. This function does not have a return keyword.